fairscale/optim/oss.py

```diff
@@ -247,6 +248,7 @@ def clip_grad_norm(
         Arguments:
             max_norm (float or int): max norm of the gradients
             norm_type (float or int): type of the used p-norm. Can be ``'inf'`` for infinity norm.
+            skip_nan_grads (bool): auto-skip scaling if some grads are NaNs (which can happen with Pytorch AMP, in that case the model update is skipped)
```

Comment:
if NaN grads are ok (basically if you're using torch AMP), then this can be passed